app/subscriber/src/components/sidebar/advanced-search/AdvancedSearch.tsx

```diff
@@ -101,6 +110,21 @@ export const AdvancedSearch: React.FC<IAdvancedSearchProps> = ({
     [advancedSearch, constants?.frontPageId],
   );
 
+  const saveSearch = React.useCallback(async () => {
+    const data = await addFilter({
```

Comment:
Looks like a unhandled promise to me.  Testing with the developer console normally shows an error for this stuff.